src/subscript/pack_sim/pack_sim.py

```diff
@@ -420,50 +434,72 @@ def inspect_file(
-            section = "regions/"
-            if not os.path.exists("%s/include/%s" % (packing_path, section)):
-                os.makedirs("%s/include/%s" % (packing_path, section))
+        elif line_strip == "REGIONS" and fmu_include:  # type: ignore
+            section = "regions/"  # type: ignore
+            if not os.path.exists(
+                "%s/include/%s" % (packing_path, section)  # type: ignore
+            ):
+                os.makedirs("%s/include/%s" % (packing_path, section))  # type: ignore
             new_data_file += line
-        elif line_strip == "SOLUTION" and fmu_include:
-            section = "solution/"
-            if not os.path.exists("%s/include/%s" % (packing_path, section)):
-                os.makedirs("%s/include/%s" % (packing_path, section))
+        elif line_strip == "SOLUTION" and fmu_include:  # type: ignore
+            section = "solution/"  # type: ignore
+            if not os.path.exists(
+                "%s/include/%s" % (packing_path, section)  # type: ignore
+            ):
+                os.makedirs("%s/include/%s" % (packing_path, section))  # type: ignore
             new_data_file += line
-        elif line_strip == "SUMMARY" and fmu_include:
-            section = "summary/"
-            if not os.path.exists("%s/include/%s" % (packing_path, section)):
-                os.makedirs("%s/include/%s" % (packing_path, section))
+        elif line_strip == "SUMMARY" and fmu_include:  # type: ignore
+            section = "summary/"  # type: ignore
+            if not os.path.exists(
+                "%s/include/%s" % (packing_path, section)  # type: ignore
+            ):
+                os.makedirs("%s/include/%s" % (packing_path, section))  # type: ignore
             new_data_file += line
-        elif line_strip == "SCHEDULE" and fmu_include:
-            section = "schedule/"
-            if not os.path.exists("%s/include/%s" % (packing_path, section)):
-                os.makedirs("%s/include/%s" % (packing_path, section))
+        elif line_strip == "SCHEDULE" and fmu_include:  # type: ignore
+            section = "schedule/"  # type: ignore
+            if not os.path.exists(
+                "%s/include/%s" % (packing_path, section)  # type: ignore
+            ):
+                os.makedirs("%s/include/%s" % (packing_path, section))  # type: ignore
             new_data_file += line
-        elif line_strip == "OPTIMIZE" and fmu_include:
-            section = "optimize/"
-            if not os.path.exists("%s/include/%s" % (packing_path, section)):
-                os.makedirs("%s/include/%s" % (packing_path, section))
+        elif line_strip == "OPTIMIZE" and fmu_include:  # type: ignore
+            section = "optimize/"  # type: ignore
+            if not os.path.exists(
+                "%s/include/%s" % (packing_path, section)  # type: ignore
+            ):
+                os.makedirs("%s/include/%s" % (packing_path, section))  # type: ignore
```

Comment:
What's going on here? And could this maybe be better handled by ignoring the file in a mypy config?

Reply:
You've found the black sheep in subscript.. `pack_sim` is in need for a general rewrite yes, (see #363). I struggled a bit with this part but found no better way than ignoring these lines. Ignoring the whole file is perhaps a little bit too permissive?

Reply:
This is cluttering the history, so I lean towards ignoring the whole file.

Reply:
Will fix 👍🏻